test/functional/rpc_fundrawtransaction.py

```diff
@@ -1143,6 +1144,21 @@ def test_feerate_rounding(self):
         rawtx = w.createrawtransaction(inputs=[], outputs=[{self.nodes[0].getnewaddress(address_type="bech32"): 1 - 0.00000202}])
         assert_raises_rpc_error(-4, "Insufficient funds", w.fundrawtransaction, rawtx, {"fee_rate": 1.85})
 
+    def test_sffo_with_negative_ev(self):
+        self.log.info("Testing that subtract fee from outputs with negative effective value outputs still works")
+
+        self.nodes[1].createwallet("negevsffotest")
+        w = self.nodes[1].get_wallet_rpc("negevsffotest")
+
+        self.nodes[0].sendtoaddress(w.getnewaddress(address_type="bech32"), 0.00001000)
+        self.nodes[0].sendtoaddress(w.getnewaddress(address_type="bech32"), 1)
+
+        self.generate(self.nodes[0], 1)
+
+        created = w.createrawtransaction([], [{self.nodes[0].getnewaddress(address_type="bech32"): 1.00001000}])
+
+        # With 15 sat/vb, the low value input is negative ev. This should still fund.
+        funded = w.fundrawtransaction(created, {"subtractFeeFromOutputs":[0], "fee_rate": 15})
```

Comment:
`test/functional/rpc_fundrawtransaction.py:1161:9: F841 local variable 'funded' is assigned to but never used`

Reply:
Fixed.